recipes/seamm-datastore/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "seamm-datastore" %}
+{% set version = "2021.11.21" %}
+{% set sha256 = "ac0af62629dba57e92f81861981eef9bf8557cdca00d5f443e4022c5e510aeb0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/seamm-datastore/seamm_datastore-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools
+  run:
+    - flask
+    - flask-authorize
+    - python >=3.8
+    - python-dateutil
+    - sqlalchemy
+
+test:
+  imports:
+    - seamm_datastore
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/molssi-seamm/seamm-datastore
```

Comment:
This link appears to be incorrect. Should be https://github.com/molssi-seamm/seamm_datastore

Reply:
Thanks! Dashes and underscores will be the death of me! The PyPi links are particularly fun in this regard.